Regenerate bindings with name

gsuuon opened the pull request:
fixes #21 

This removes the generate action in `binding.gyp` and regenerates the binding files. I guess the last committed set of generated files were missing a name in `grammar.js`?

Nsidorenco commented:
Hi,
Thank you for taking the time to play around with this.

I removed the pre-generated parser from the repo and added the generate action due to it causing unnecessary merge conflicts while the grammar is still actively developed.

I am not too keen on adding it back just yet, but I do plan on publishing the generated parser files in a release when I get around to it.

vzarytovskii commented:
> I am not too keen on adding it back just yet, but I do plan on publishing the generated parser files in a release when I get around to it. 

I can play around with automated publishing to tags/separate branch/releases on every merge to main, if you'd like to?

Nsidorenco commented:
@vzarytovskii if you have the time for it you're more than welcome!

Nsidorenco commented:
@gsuuon Good catch on the missing names in the binding files through! I'll get those fixed

gsuuon commented:
> Hi, Thank you for taking the time to play around with this.
> 
> I removed the pre-generated parser from the repo and added the generate action due to it causing unnecessary merge conflicts while the grammar is still actively developed.
> 
> I am not too keen on adding it back just yet, but I do plan on publishing the generated parser files in a release when I get around to it.

The parsers were still in the repo so I just included them - the main change here (besides the missing name) was getting rid of the build action in `binding.gyp` which is breaking `:TSInstallFromGrammar` on windows. It doesn't seem like that build action is necessary to get this working with nvim-treesitter [based on this](https://github.com/nvim-treesitter/nvim-treesitter#adding-parsers) (`You can also skip step 2 and use :TSInstallFromGrammar zimbu to install directly from a grammar.js in the top-level directory specified by url`) - the parser is only expected to already be there if we use `:TSInstall`
